bitwatch/docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: bitwatch_web_1
+      APP_PORT: 3117
+
+  web:
+    image: ghcr.io/zapomatic/bitwatch:v1.4.4@sha256:12e8326db0e5027a24dec723fb849178e4687464705351d7e6d7d42068ca96eb
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/data:/app/server/data
```

```suggestion
      - ${APP_DATA_DIR}/data:/app/server/data

```
Add a final new line here.